meetings/2023-01-04.md

```diff
@@ -0,0 +1,128 @@
+  * James, web platform compatibility question
+    * globalThis extends EventTarget in browsers/deno
+    * cons
+      * key argument against it is that we use process to emit events, second path could
+        cause confusion.
+      * It’s a breaking change on its own
+      * If browser developers were to do it again feedback is that they would not do it again
+    * Pro is web compatibility
+  * Geoffrey
+    * Any way to have toggle on/off
+    * James, in my opinion likely does not add enough value
+    * Joyee, would add complexity for module authors as well
+  * James, have discussed in WinterCG if to make globalThis part of minimal
+    requirements, decided not to.
+  * Tobias, trying to find out the value of doing it.
+    * background, its related to WebWorker compatibility, would be a requirement for that.
+  * James, cloudflare workers is trying to move away from the model that requires globalThis
+    being event target.
+  * James, web worker implementation could provide a globalThis that does extend EventTarget
+  * Ronag, would we make registering on globalThis just register on process. Likely not since
+    events methods get a this.
+  * At this point the consensus seems to be that value does not outway the issues/problems
+  * Is there anything that is impossible because of this would be the last question to answer
+    * Antoine - dispatching custom events on globalThis
+    * James already possible use vm module to run with globalThis that extends EventTarget
+    * Matteo raised concerns on brittleness of the pattern as well
+  * Comment on issue
+    We discussed in the TSC meeting today, at this point the consensus seems to be that value
+    does not outway the issues/problems unless there is a compelling use case we are not aware
+    of. We understand it could be useful for some cases (like webworkers) case but feel there are
+    workarounds.
+  * Geoffrey
+    * I’ll open a PR on the `node` repo to update  
+      <https://github.com/nodejs/node/blob/main/doc/api/documentation.md> to list the new
+      subdivisions within “experimental” proposed here. In this first PR I’ll leave out any discussion
+      of flags or warnings.
+    * Once ☝️ lands, I’ll open a follow-up PR adding requirements for each level, such as flags or
+      warnings.
+    * Concurrent with the second PR, we need to review the docs and update all the currently
+      experimental features to distinguish which experimental level they are. I don’t anticipate us
+      adding any flags or warnings to experimental features that currently lack them, but we can
+      consider doing so if we think they might be warranted.
+    * Once the above is complete, this would become the process for new experimental features
+      going forward, that would be considered the “default” process unless we decide to grant
+      exceptions on a case-by-case basis.
```

Comment:
All of this belongs below the “Proposal: Subdivide “Experimental” Status” section.

Reply:
I think fixed.